src/pages/Authentication/components/Input/Input.tsx

```diff
@@ -38,20 +22,6 @@ export const Input = memo((props: InputPropsType) => {
     }
   }[chain];
 
-  const handleGenerateClick = async () => {
-    const route = search
-      ? `${window.location.origin}${routeNames.unlock}${search}&callbackUrl=${callbackRoute}`
-      : `${window.location.origin}${routeNames.unlock}?callbackUrl=${callbackRoute}`;
-    const isWallet = loginMethod === 'wallet';
-    const redirect = isWallet ? encodeURIComponent(route) : route;
-
-    await logout(redirect);
-  };
-
-  /*
-   * Return the rendered component.
-   */
-
   return (
     <Formik
       onSubmit={() => {}}
```

Comment:
?

Reply:
onSubmit it's mandatory, but don't want to do something with this method